### PR TITLE
Scaffold: Obras

### PR DESCRIPTION
**Pull request by varffs**
Outstanding:
- Artist select
- Translations

### CODE REVIEW

File: src/components/obras/ObraForm.jsx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+          </div>
+        </div>
+
+        <div className='grid-row margin-bottom-basic'>
+          <div className='grid-item item-s-12'>
+            <h4 className='font-size-small font-bold margin-bottom-tiny'><label htmlFor='artista'>Artista</label></h4>
+            <input
+              id='artista'
+              name='artista'
+              type='text'
```
Comment (mikeyrayvon) [inline src/components/obras/ObraForm.jsx]: this should be a select field filled with artists: id / name as key / value pair

Comment (varffs) [reply src/components/obras/ObraForm.jsx]: yh it says in the PR artist select is outstanding

Comment (mikeyrayvon) [reply src/components/obras/ObraForm.jsx]: oic

**Comment by mixcas** [comment]
I added the basic Artista select. Is missing some things, like passing down: `id`, `name` and `disabled`. Could u do that @varffs ?

**Comment by varffs** [comment]
Not sure what you were thinking for `disabled` (why would it be?) so have added the value display on the select element and I think this is ready

**Comment by mixcas** [comment]
maybe it was `enabled` not `disabled`, maybe more like 'active'. Not necessary right now, would be useful next year when artists change. Ok like this for now.

File: src/components/obras/ObraForm.jsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+              />
+            </div>
+          </div>
+        </div>
+
+        <div className='grid-row margin-bottom-basic'>
+          <div className='grid-item item-s-12'>
+            <h4 className='font-size-small font-bold margin-bottom-tiny'><label htmlFor='artista'>Artista</label></h4>
+            <ArtistaSelectContainer value={this.state.artista} onChange={this.handleArtistaChange} />
+            <input
```
Comment (mixcas) [inline src/components/obras/ObraForm.jsx]: We don't need this so i'm going to remove it

**Comment by varffs** [comment]
Pretty sure the artist hidden field is the place the data is saved? Does this still save that data to the Obra?

**Comment by mixcas** [comment]
@varffs  https://github.com/interglobalvision/admin-andamiaje-co/pull/12/files#diff-719464f7694c31ccd24c7b938d574b75R78

**Pull request merged**